src/main/java/uk/gov/hmcts/ccd/domain/model/definition/DynamicListItem.java

```diff
@@ -0,0 +1,4 @@
+package uk.gov.hmcts.ccd.domain.model.definition;
+
+public class DynamicListItem extends FixedListItem{
```

Comment:
missing space FixedListItem{

Reply:
removed this class now as per discussion